vermouth/rcsu/go_structure_bias.py

```diff
@@ -177,22 +179,28 @@ def contact_selector(self, molecule):
                     # find the go virtual-sites for this residue
                     # probably can be done smarter but mehhhh
                     atype_a = next(get_go_type_from_attributes(self.res_graph.nodes[resA]['graph'],
-                                                               resid=resIDA,
+                                                               _old_resid=resIDA,
                                                                chain=chainA,
                                                                prefix=self.moltype))
                     atype_b = next(get_go_type_from_attributes(self.res_graph.nodes[resB]['graph'],
-                                                               resid=resIDB,
+                                                               _old_resid=resIDB,
                                                                chain=chainB,
                                                                prefix=self.moltype))
+                    # Check if symmetric contact has already been processed before
+                    # and if so, we append the contact to the final symmetric contact matrix
+                    # and add the exclusions. Else, we add to the full valid contact_matrix
+                    # and continue searching.
```

Comment:
This comment needs to be expended to explain the /why/. Prefferably with a reference to an article.